src/main/java/com/ironcorelabs/tenantsecurity/kms/v1/TenantSecurityKMSRequest.java

```diff
@@ -68,49 +71,56 @@ public void close() throws IOException {
     }
 
     /**
-     * Build up a generic HttpRequest which includes the appropriate headers to make
-     * requests to the Tenant Security Proxy.
+     * Build up a generic HttpRequest which includes the appropriate headers to make requests to the
+     * Tenant Security Proxy.
      */
-    private HttpRequest getApiRequest(Map<String, Object> postData, GenericUrl endpoint) throws Exception {
-        return requestFactory.buildPostRequest(endpoint, new JsonHttpContent(JSON_FACTORY, postData))
+    private HttpRequest getApiRequest(Map<String, Object> postData, GenericUrl endpoint)
+            throws Exception {
+        return requestFactory
+                .buildPostRequest(endpoint, new JsonHttpContent(JSON_FACTORY, postData))
                 // Clone the headers on use. Otherwise Google will keep appending their custom
                 // user agent string and it will grow big enough to cause header overflow
                 // errors.
-                .setHeaders(this.httpHeaders.clone())
-                .setReadTimeout(this.timeout)
+                .setHeaders(this.httpHeaders.clone()).setReadTimeout(this.timeout)
```

Comment:
I don't understand why these lines were combined. Seems odd.

Reply:
I agree. I couldn't figure out why the formatter was going crazy on files in general, but specifically the DocumentMetadata one. I tested it on other files and it didn't need to make any changes, but for some reason it really didn't like what was going on in that file.